_includes/v2.0/prod-deployment/secure-start-nodes.md

```diff
@@ -138,6 +138,7 @@ For each initial node of your cluster, complete the following steps:
      Flag | Description
      -----|------------
      `--join` | Identifies the address and port of 3-5 of the initial nodes of the cluster.
+     `--advertise-host` | The hostname or IP address to advertise to other CockroachDB nodes. If it is a hostname, it must be resolvable from all nodes; if it is an IP address, it must be routable from all nodes.
```

Comment:
I'm in the process of updating this and other text around `--advertise-host` in a separate PR. I can just resolve the conflict on my end, or you can revert this change. Your call.

Reply:
IIRC, we decided that I will include the --advertise-host description for the systemd section, and you will update the manual process. Does your PR take care of both? If yes, I will happily revert my change.

Reply:
Yes, my PR will cover both, so you can revert this.